frontend/src/App.tsx

```diff
@@ -99,6 +100,12 @@ const App = (): React.ReactElement => {
                   path={Routes.EMAIL_ACTION}
                   component={EmailActionHandler}
                 />
+                <PrivateRoute
```

Comment:
can you add a quick comment?
```suggestion
// TODO: Remove route for production
                <PrivateRoute
```